game-core/src/main/java/games/strategy/engine/framework/startup/mc/ServerModel.java

```diff
@@ -633,8 +638,6 @@ public void connectionRemoved(final INode node) {
       serverLauncher.connectionLost(node);
       return;
```

Comment:
This return statement was the culprit, for in progress games `serverLauncher != null`